### PR TITLE
https://issues.redhat.com/browse/ACM-10649

### PR DESCRIPTION
**Pull request by dockerymick**

### CODE REVIEW

File: governance/compliance_history.adoc [hunk 1]
```diff
@@ -67,7 +68,8 @@ python -c 'import urllib.parse; import sys; print(urllib.parse.quote(sys.argv[1]
 [source,bash]
 ----
 curl -H "Authorization: Bearer $(oc whoami --show-token)" \
-    "https://$(oc -n open-cluster-management get route governance-history-api -o jsonpath='{.spec.host}')/api/v1/compliance-events"
+    "https://$(oc -n open-cluster-management get route governance-history-api -o
+jsonpath='{.spec.host}')/api/v1/compliance-events"
```
Comment (mprahl) [inline governance/compliance_history.adoc]: Why did this line change?

File: governance/compliance_history.adoc [hunk 2]
```diff
@@ -168,4 +171,4 @@ Any new policy compliance events are recorded in the policy compliance history A
 [#add-compliance-history-resources]
 == Additional resource
```
Comment (mprahl) [inline governance/compliance_history.adoc:171]: Since this is a list, should resource be plural?

Comment (dockerymick) [reply governance/compliance_history.adoc:171]: Well there is only one resource in the list so far, so it accurate to state additional resource. Is there another link that should be added?

File: governance/compliance_history.adoc (hunk 1 above)
Comment (dockerymick) [reply governance/compliance_history.adoc]: When you mentioned indents, I assumed that you were referring to the command. I did see the misalignment though and reverted my change.

**Review by mprahl — APPROVED** [review]

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**